docroot/modules/custom/foia_migrate/config/install/migrate_plus.migration.agency_component.yml

```diff
@@ -171,7 +182,7 @@ process:
   field_submission_address/administrative_area: state
   field_submission_address/postal_code: zip
   field_submission_fax: fax
-  field_submission_email: email_submission
+  field_submission_email: email
```

Comment:
So this is a little tricky because I _think_ there was one agency component where I actually manually put an email address in an `email_submission` key in the source data. We'll have to change that component manually to move that email address to an `emails` key instead, so that it is picked up here.